pkgs/applications/networking/browsers/ladybird/default.nix

```diff
@@ -30,6 +31,18 @@ in gcc11Stdenv.mkDerivation {
     hash = "sha256-XQj2Bohk8F6dGCAManOmmDP5b/SqEeZXZbLDYPfvi2E=";
   };
 
+  postPatch = ''
+    mkdir serenity
+    cp -r ${serenity}/* serenity
+    chmod -R +w serenity
+  ''
+  # https://github.com/SerenityOS/serenity/issues/10055
+  + lib.optionalString stdenv.isDarwin ''
+    sed -i '/MACOSX_BUNDLE/d' CMakeLists.txt
```

Comment:
Why do we delete the `MACOSX_BUNDLE` line? Does it have to do with the rpath issue?

Reply:
The output `$out/bundle/Ladybird.app` appears to be corrupted. I removed these lines so as not to bundle the package, i.e. the binaries reside in `$out/bin` like on Linux.

Reply:
Then can you move the comment about the issue below the sed call so that it's clear that it only refers to the rpath patch?

Reply:
Done.